pkgs/by-name/wc/wcurl/package.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv
```

Comment:
Maybe use the newer nixpkgs formatting style, no a requirement but would be nice

Reply:
I ran nixfmt-rfc-style, so hopefully this is correct now.